use-location.js

```diff
@@ -1,5 +1,10 @@
 import { useEffect, useRef, useState, useCallback } from "./react-deps.js";
 
+const eventPopState = "popstate"
+const eventPushState = "pushstate"
```

Comment:
```suggestion
const eventPushState = "pushState"
```

The `eventPushState` as a name of a method when performing a navigation with `history.pushState()` and `history.replaceState()`

Reply:
My bad, fixed it and added a comment for others to quickly check MDN, if additions/changes would be made